packages/ui-react/src/components/Card/Card.tsx

```diff
@@ -95,6 +95,8 @@ function Card({
     }
   };
 
+  const heading = React.createElement(`h${headingLevel}`, { className: classNames(styles.title, { [styles.loading]: loading }) }, title);
```

Comment:
The dynamic heading level was only applied once previously. So I also added this as a bonus

Reply:
We could make a `<Heading />` component actually, to be used anywhere. (maybe not now)